docs/framework/data/adonet/ef/language-reference/csdl-ssdl-and-msl-specifications.md

```diff
@@ -4,6 +4,5 @@ ms.date: "03/30/2017"
 ms.assetid: bbdc9237-ff4c-4441-9565-31ebc29743e9
 ---
 # CSDL, SSDL, and MSL Specifications
```

Comment:
I redirected to https://docs.microsoft.com/ef/ef6/index, since that's what the MSDN link pointed to. There are three different possibilities for redirect targets, so I don't know if there's a better way to handle it.

This file was a TOC parent to three other pages (including the other one you wanted to delete), so I deleted and set up redirects for four articles in total.